spark-print-data/printertypes.json

```diff
@@ -656,10 +656,100 @@
                 "z_axis_feed_rate": 3
             },
             "_files": [
-				"icons.16x16_id",
-				"icons.32x32_id",
-				"icons.50x50_id",
-				"icons.100x100_id",
+                "icons.16x16_id",
+                "icons.32x32_id",
+                "icons.50x50_id",
+                "icons.100x100_id",
+                "build_volume.bed_file_id"
+            ]
+        }, 
+        {
+            "id" : "D4H4F6G4-8J4H-5F6K-4F3J-4GK5FH4H56GF",
+            "version" : 1,
+            "name" : "DreamMaker_OverLord",
+            "manufacturer" : "DFRobot",
+            "registration_url" : null,
+            "model_number" : "1.0.0",
+            "icons" : {
+                "16x16_id" : "data/DreamMaker_OverLord_16x16.png",
+                "100x100_id" : "data/DreamMaker_OverLord_100x100.png",
+                "200x60_id" : "data/DreamMaker_OverLord_200x60.png",
+                "400x120_id" : "data/DreamMaker_OverLord_400x120.png",
+                "50x50_id" : "data/DreamMaker_OverLord_50x50.png",
+                "32x32_id" : "data/DreamMaker_OverLord_32x32.png"
+            },
+            "technology" : "FDM",
+            "default_material_id": "B27DE79C-0459-4B59-87FD-3093313126EF",
+            "default_profile_id": "F3314255-711D-48A9-8CB2-C55CEACF58B7",
+            "firmware": {
+                "type" : "Marlin",
+                "version" : "2.5.4"
+            },
+            "build_volume": {
+                "type": "Cartesian",
+                "bed_size": [
+                    17,
+                    17,
+                    26
+                ],
+                "bed_offset": [
+                    0,
+                    0,
+                    0
+
+                ],
+                "home_position": [0, 0, 0],
+                "park_position": [0, 0, 250],
+                "bed_file_id": "data/DreamMaker_OverLord.zip"
+            },
+            "max_materials": 1,
+            "printable": {
+                "content": "application/gcode",
+                "thumbnail": "none",
+                "extension": "gcode",
+                "generates_supports": false
+            },
+            "supported_connections": [
+                {
+                    "type": "serial",
+                    "discovery": "serial",
+                    "protocol": "usb.serial",
```

Comment:
Are you going to supply VID/PID for these connections https://github.com/spark3dp/print-manager/blob/master/spark-print-mgr/printers/drivers/driverConfig.json